src/Psl/Xml/Internal/issue_from_xml_error.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml\Internal;
+
+use LibXMLError;
+use Psl\Xml\Issue\Issue;
+
+/**
+ * @psalm-internal \Psl\Xml
```

Comment:
```suggestion
 * @psalm-internal
 *
 * @psalm-pure
```

Reply:
Doesnt that mean it cannot be used outside `Psl\Xml\Internal`?

Reply:
yes, but not outside `Psl`

Reply:
https://psalm.dev/docs/annotating_code/supported_annotations/#psalm-internal

Psalm needs me to provide a namespace. I'dd say `Psl\Xml` is ok, since other parts of psl don't have anything to do with xml?

Reply:
not sure how we do this everywhere else, but we don't usually supply the namespace.

Reply:
let's go with @internal for consistency ( https://github.com/azjezz/psl/blob/develop/src/Psl/Internal/Loader.php#L25 )